kafka/adminapi.go

```diff
@@ -148,7 +148,7 @@ func ResourceTypeFromString(typeString string) (ResourceType, error) {
 	case "BROKER":
 		return ResourceBroker, nil
 	default:
-		return ResourceUnknown, newGoError(ErrInvalidArg)
+		return ResourceUnknown, NewError(ErrInvalidArg, "unknown resource type", false)
```

Comment:
Not sure about error string here, what do you think?

Reply:
Looks good, but capitalize first character: "Unknown .."